test/FunctionalTests/HealthChecks.RavenDB/RavenDBWithSingleConnectionStringHealthCheckTests.cs

```diff
@@ -13,12 +13,12 @@
 namespace FunctionalTests.HealthChecks.RavenDB
 {
     [Collection("execution")]
-    public class ravendb_healthcheck_should
+    public class ravendb_with_single_connection_string_healthcheck_should
```

Comment:
Well, if you use only one constructor, probably create tests for one or more connectionstring doens't have sense, true?

Reply:
HI @unaizorrilla I realized that we would still need the separate tests since we still have two method overloads in `RavenDBHealthCheckBuilderExtensions` even though the constructor is one.